src/registrar/assets/src/sass/_theme/_typography.scss

```diff
@@ -23,6 +23,14 @@ h2 {
   color: color('primary-darker');
 }
 
+.header--body {
```

Comment:
**non blocking question** can you explain the switch to header-- body?

Reply:
Ah yes. The old CSS class identified those style headers as a register form header, semantically (screen readers will read classes I think): register-form-review-header

The Figma for this ticket introduces a header variant on pages other than the register form that look similar to those old headers on the register form. SO, I created a more generic CSS class called header--body that indicates that the header is a variation of type --body (the font size matches the `<p>` font size. Not the best CSS class name but I couldn't think of something better.

Reply:
Oh that makes sense! Thanks for the explanation, I wouldn't have guessed that otherwise! resolving this convo.